### ISSUE TITLE
Error while running patched game with wine

### ISSUE THREAD

**Issue by wdmvv** [thpatch/thcrap#189]
wine version: wine-6.0.3 (Ubuntu 6.0.3~repack-1)
.net - 4.6.1
directx is installed

what is my problem: if I try to run patched game/config.exe I get error - 
```
wine: Call from 7B010BE6 to unimplemented function api-ms-win-crt-convert-l1-1-0.dll.mbrtoc16, aborting
wine: Unimplemented function api-ms-win-crt-convert-l1-1-0.dll.mbrtoc16 called at address 7B010BE6 (thread 0378), starting debugger...
```

This is probably not wine problem since I can run game in japanese without any problems
reference image -
![image](https://user-images.githubusercontent.com/78714307/176117421-98ae3be5-cfc7-4320-b760-00e7d1700f4a.png)

full wine log: [backtrace.txt](https://github.com/thpatch/thcrap/files/8998558/backtrace.txt)

**Comment by sin-ack** [comment]
Thanks, replacing the files worked.

**Comment by DankRank** [comment]
this is fixed in commit d7af05bbb421886bfd1e6adb5657ba640897a376
